GitFyle.Core.Api/Models/Foundations/Configurations/Configuration.cs

```diff
@@ -13,8 +13,8 @@ public class Configuration
         public string Value { get; set; }
         public string Type { get; set; }
         public string CreatedBy { get; set; }
-        public DateTime CreatedDate { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
         public string UpdatedBy { get; set; }
-        public DateTime UpdatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
     }
 }
```

Comment:
Good catch. You will need a new migration for the changes as well.

Reply:
Migration added.